mypy/binder.py

```diff
@@ -8,6 +8,12 @@
 from mypy.join import join_simple
 from mypy.sametypes import is_same_type
 
+from mypy.nodes import IndexExpr, MemberExpr, NameExpr
+
+
+BindableTypes = (IndexExpr, MemberExpr, NameExpr)
+BindableExpression = Union[IndexExpr, MemberExpr, NameExpr]
+
```

Comment:
I don't see the point of introducing this `BindableExpression`. It seems to duplicate information with which `Expression`s have (non-`None`) `literal`s. It should be fine to just use `Expression` everywhere.